spec.html

```diff
@@ -409,7 +409,7 @@ <h1>Number value</h1>
 
     <emu-clause id="sec-terms-and-definitions-number-type">
       <h1>Number type</h1>
-      <p>set of all possible Number values including the special “Not-a-Number” (NaN) value, positive infinity, and negative infinity</p>
+      <p>set of all possible Number values including the special NaN (“not a number”) value, positive infinity, and negative infinity</p>
```

Comment:
```suggestion
      <p>set of all possible Number values including the special *NaN* (“not a number”) value, positive infinity, and negative infinity</p>
```

(my fault for not putting it in my earlier suggestion, but now that we're referring to NaN as the value itself rather than giving it as the acronym it's appropriate to bold I think)